packages/core/src/Stores/ui-store.js

```diff
@@ -173,6 +173,9 @@ export default class UIStore extends BaseStore {
     is_cfd_reset_password_modal_enabled = false;
     sub_section_index = 0;
 
+    is_additional_kyc_info_modal_open = false;
+    is_information_submitted_modal_open = false;
```

Comment:
can we rename this to is_kyc_information_submitted_modal_open or something more specific to kyc modal?

Reply:
Yeah sure. Will update 👍